BaseApi.js

```diff
@@ -0,0 +1,65 @@
+const baseUrl = 'https://aplus-dev.liqd.net/api'
+
+const endpoints = {
+  ideas: baseUrl + '/modules/$moduleId/ideas',
+  liveQuestions: baseUrl + '/modules/$moduleId/interactiveevents/livequestions/'
+}
+
+const makeGetRequest = (url) => {
+    return fetch(url)
+      .then((response) => response.json())
+      .then((unpackedData) => {
```

Comment:
oh just saw i left this unnecessary line break (probably when trying something).
Here you could make a one line as the line above.

Reply:
Will remove it then!

Reply:
Hm, thats giving me an error `Unexpected token` :woman_shrugging: So I'll leave it like this!